eng/pipelines/templates/variables/globals.yml

```diff
@@ -7,4 +7,4 @@ variables:
   ServiceVersion: ""
   Package.EnableSBOMSigning: true
   # Disable warning until issue 21765 and 21766 are closed
-  DisableDockerDetector: true
```

Comment:
why did this file change?

Reply:
I am thinking use global variable. But the platform-matrix did not like this idea.